### ISSUE TITLE
Container is unhealthy !?

### ISSUE THREAD

**Issue by basselissa** [Sitecore/docker-images#297]
![image](https://user-images.githubusercontent.com/14176102/80224129-576a8a00-865a-11ea-9a0d-efea61d5b643.png)

**Comment by anarsen** [comment]
Did you set the Sitecore license environment variable? If not, the container will never become healthy.

See this section: https://github.com/Sitecore/docker-images#quick-start under _For Sitecore 9.3.x_.

**Comment by basselissa** [comment]
I did, running the license environment variable never show any error. but i don't think that it is creating any env variable. do i need PS 6 or 7 ?

**Comment by basselissa** [comment]
Dears, I tried all possibilities,  and i am sure that environment license is loaded correctly 
but still getting this error 
![image](https://user-images.githubusercontent.com/14176102/80485771-12e93200-896b-11ea-920d-cb31430929c6.png)

**Comment by basselissa** [comment]
here is the env:* i have right after I run the set license environment variable script
![image](https://user-images.githubusercontent.com/14176102/80486510-57c19880-896c-11ea-910f-bd61b283dc1b.png)

what could be missing?

**Comment by pbering** [comment]
Is this related to your other issue #290 ?

**Comment by alan-null** [comment]
Hey @basselissa @pbering 
I encountered the same problem today.

Additional info from my side

I can bring up this
`docker-compose.xm.yml`
and it works fine (so I would eliminate problem with license)

for these:
`docker-compose.xp.sxa.yml`
`docker-compose.xp.spe.yml`

I get
```
ERROR: for cd  Container "efb10bf844a1" is unhealthy.
ERROR: Encountered errors while bringing up the project.
```
```
PS C:\docker-images\windows\tests\9.3.x> docker-compose -f docker-compose.xp.spe.yml up -d
Creating network "93x_default" with the default driver
Creating 93x_sql_1  ... done
Creating 93x_solr_1 ... done
Creating 93x_xconnect_1 ... done
Creating 93x_cm_1                        ... done
Creating 93x_xconnect-processingengine_1 ... done
Creating 93x_cd_1                        ... done
Creating 93x_xconnect-automationengine_1 ... done

ERROR: for xconnect-indexworker  Container "2b94d25bd257" is unhealthy.
ERROR: Encountered errors while bringing up the project.
PS C:\docker-images\windows\tests\9.3.x> docker container ls
CONTAINER ID        IMAGE                                                                    COMMAND                   CREATED              STATUS              PORTS                     NAMES
d442a20b80bb        sitecore-xp-xconnect-automationengine:9.3.0-windowsservercore-ltsc2019   "powershell.exe -Com…"    43 seconds ago       Up 30 seconds                                 93x_xconnect-automationengine_1
2778894c3fe4        sitecore-xp-cd:9.3.0-windowsservercore-ltsc2019                          "powershell.exe -Com…"    43 seconds ago       Up 28 seconds       0.0.0.0:44002->80/tcp     93x_cd_1
f02a660b07a3        sitecore-xp-xconnect-processingengine:9.3.0-windowsservercore-ltsc2019   "powershell.exe -Com…"    43 seconds ago       Up 29 seconds                                 93x_xconnect-processingengine_1
2d12282a3cb7        sitecore-xp-spe-standalone:9.3.0-windowsservercore-ltsc2019              "powershell.exe -Com…"    43 seconds ago       Up 30 seconds       0.0.0.0:44001->80/tcp     93x_cm_1
0b241460910d        sitecore-xp-xconnect:9.3.0-windowsservercore-ltsc2019                    "C:\\ServiceMonitor.e…"   48 seconds ago       Up 43 seconds       80/tcp                    93x_xconnect_1
2945dc332d0e        sitecore-xp-solr:9.3.0-nanoserver-1809                                   "cmd /S /C Boot.cmd …"    About a minute ago   Up About a minute   0.0.0.0:44011->8983/tcp   93x_solr_1
PS C:\Users\alan\sc-docker\docker-images\windows\tests\9.3.x>
```
I haven't had time to investigate it. I just wanted to check how things work. 
By the way I am using Azure VM (fresh, created just for tests)

**Comment by pbering** [comment]
Closing this one for now, please create new issue with you still have problems after trying fix #312
